### PR TITLE
Generate GuestId For Non Registered Users

### PR DESCRIPTION
**Pull request by alexisljn**
| Q             | A
| ------------- | ---
| Is bugfix?    | No
| New feature?  | Yes
| Fixed issues  | Fixes

Generates a unique guest Id for guest users.

### CODE REVIEW

**Comment by alexisljn** [comment]
Review Required !

File: backend/util/GameUtil.js [hunk 1]
```diff
@@ -2,13 +2,13 @@
 class GameUtil {
 
     static ROOMS_ID = [];
+    static GUEST_ID = [];
```
Comment (EmileCalixte) [inline backend/util/GameUtil.js:5]: It is an array, shouldn't it be called "GUEST_IDS" ?

**Review by EmileCalixte — CHANGES_REQUESTED** [review]

File: backend/util/GameUtil.js (hunk 1 above)
Comment (alexisljn) [reply backend/util/GameUtil.js:5]: You're right

**Comment by alexisljn** [comment]
Review Required

**Review by EmileCalixte — APPROVED** [review]
Super !

**Pull request merged**